### PR TITLE
Adds ui elements for passwordless login

### PR DESCRIPTION
**Pull request by timbastin**

### CODE REVIEW

File: src/components/common/Carriage.tsx
Comment (seb-kw) [inline src/components/common/Carriage.tsx:1]: I know this is rather annoying, but where does the SVG come from (license issue etc.)?

Comment (timbastin) [reply src/components/common/Carriage.tsx:1]: https://media.istockphoto.com/id/963841644/de/vektor/vintage-pferdekutsche.jpg?s=612x612&w=0&k=20&c=nM6soHd0dBWaC5TVSLzXrnXLSMqQftPNZJzYPryDkPk=

Comment (timbastin) [reply src/components/common/Carriage.tsx:1]: https://www.istockphoto.com/de/vektor/vintage-pferdekutsche-gm963841644-263195929?searchscope=image%2Cfilm

Comment (seb-kw) [reply src/components/common/Carriage.tsx:1]: Seems we can claim their standard license. Nothing written about credit requirements or something. So seems fine.

**Review by seb-kw — APPROVED** [review]

**Pull request merged**